errors/native/src/lib.rs

```diff
@@ -18,38 +16,34 @@ fn throw_error(mut cx: FunctionContext) -> JsResult<JsString> {
 }
 
 /// Throw TypeError
-fn throw_type_error(mut cx: FunctionContext) -> JsResult<JsValue> {
-    let foo: JsResult<JsError> = cx.throw_type_error("not enough arguments");
+fn _throw_type_error(mut cx: FunctionContext) -> JsResult<JsValue> {
+    let _foo: JsResult<JsError> = cx.throw_type_error("not enough arguments");
```

Comment:
This looks like a bug. The VM is now in a throwing state and the `cx.string` is undefined behavior. Perhaps this should have a `?` at the end?

Reply:
Is that mean people should always adding `?` at the end of throwing if they want to throw a JavaScript error, or the VM might keep running instead of terminate.